src/components/FloatingSearch.tsx

```diff
@@ -1,38 +1,152 @@
+      setShowSearchHighlight(true);
+      setSessionStorage("showSearchHighlight", true);
+    }, SEARCH_HIGHLIGHT_DELAY);
+    return () => clearTimeout(timeout);
+  }, [hasOpenedSearch]);
+
   useEffect(() => {
     setSearch(qQueryString || "");
   }, [qQueryString]);
 
+  // Clicking outside the floating search will close it
+  useEffect(() => {
+    const handleClickOutside = (event: MouseEvent) => {
+      if (ref.current && !ref.current.contains(event.target)) {
+        setShowFloatingSearch(false);
+      }
+    };
+    document.addEventListener("click", handleClickOutside, true);
+    return () => {
+      document.removeEventListener("click", handleClickOutside, true);
+    };
+  }, [ref]);
+
   return (
-    <div className="relative">
-      <form onSubmit={(e) => e.preventDefault()}>
+    <div className="relative" ref={ref}>
+      <form onSubmit={(e) => e.preventDefault()} className={`${showFloatingSearch ? "w-[425px] lg:w-[625px]" : ""}`}>
+        <button className="absolute left-0 h-full px-3 text-grey-700" onClick={() => handleSearch(search)}>
+          <span className="block">
+            <SearchIcon height="14" width="14" color="gray-500" />
+          </span>
+        </button>
         <input
           type="search"
           value={search}
+          placeholder="Full text of over 5000 climate laws and policies"
           onChange={(e) => setSearch(e.target.value)}
-          className="text-sm w-full text-indigo-400 focus:ring-0 pr-[40px]"
+          onFocus={() => handleInputFocus()}
+          className={`pl-[34px] rounded-[20px] text-sm w-full text-gray-800 border-0 focus:ring-0 focus:ring-offset-0 focus:ring-white ${
+            showFloatingSearch ? inputFocusStyles : ""
+          }`}
         />
-        <button className="absolute right-0 h-full pr-2 text-grey-700" onClick={() => handleSearch()}>
-          <span className="border-l border-lineBorder py-1 pl-2 block">
-            <SearchIcon height="20" width="20" />
-          </span>
-        </button>
       </form>
+      {showFloatingSearch && (
+        <div className="absolute bg-white px-4 pb-4 left-0 top-[36px] w-[425px] shadow-md rounded-bl-[20px] rounded-br-[20px] lg:w-[625px]">
+          <Divider />
+          <div className="text-sm mt-4">
+            <p className="text-xs text-gray-500 mb-1">Featured searches</p>
+            <ul className="flex gap-2 flex-wrap items-center">
+              {FEATURED_SEARCHES.map((searchTerm) => (
+                <li key={searchTerm}>
+                  <button
+                    onClick={() => {
+                      handleSearch(searchTerm);
+                    }}
+                    className="text-gray-800 bg-white border border-gray-300 rounded-[40px] py-1 px-2 transition hover:bg-blue-600 hover:text-white"
+                  >
+                    {searchTerm}
+                  </button>
+                </li>
+              ))}
+            </ul>
+          </div>
+          <a
```

Comment:
Maybe it was meant to be there :laughing:

Reply:
Yes it is intended.

This is a good example of what I was talking about previously regarding some basic CMS features to populate certain pieces of content.